src/compiler/scala/tools/nsc/typechecker/Typers.scala

```diff
@@ -1122,12 +1126,20 @@ trait Typers extends Adaptations with Tags with TypersTracking with PatternTyper
         }
       }
 
+      def adaptApplyInsertion(): Tree = {
+        val msg = s"The method `apply` is inserted. The auto insertion will be deprecated, please write `${tree.symbol.name}.apply` explicitly."
+        context.deprecationWarning(tree.pos, tree.symbol, msg, "2.13.13")
+        typed(Select(tree, nme.apply), mode, pt)
```

Comment:
```suggestion
        typed(atPos(tree.pos)(Select(tree, nme.apply)), mode, pt)
```

Reply:
I forgot how to make trees. Also, I never got to use `q"uotes"` much. On my other PR, I was challenged to allocate an array.